README.md

```diff
@@ -1,2 +1,15 @@
-# angular-make-our-app-di-safe-lab
-<p data-visibility='hidden'>View <a href='https://learn.co/lessons/angular-make-our-app-di-safe-lab' title='angular-make-our-app-di-safe-lab'>angular-make-our-app-di-safe-lab</a> on Learn.co and start learning to code for free.</p>
+# Making our application minification safe
+
+## Objectives
+
+- Add relevant dependency injection annotations
+
+## Instructions
+
+Let's take the code from our previous lab and change the parameter names. Change `$scope` and `$timeout` to random words of your choosing.
```

Comment:
We should provide students with starter code for this lab, rather than having them depend on a lab they've previously done. In this case, we can give them the "solution" to the previous lab with `$scope` and `$timeout` replaced with random words.